### PR TITLE
better documentation for locations, including VPC and keyPair

### PR DESCRIPTION
**Pull request by ahgittin**

### CODE REVIEW

File: guide/ops/locations/index.md [hunk 1]
```diff
@@ -62,8 +62,12 @@ brooklyn.location.jclouds.aws-ec2.credential=s3cr3tsq1rr3ls3cr3tsq1rr3ls3cr3tsq1
...
 And in this case you can reference the location in YAML with `location: jclouds:aws-ec2`.
 
-The Getting Started [template brooklyn.properties]({{ site.path.guide }}/start/brooklyn.properties) contains more examples 
-of configuring cloud endpoints, including information on credential types used in different clouds.
+Brooklyn irons out many of the differences between clouds so that blueprints run similarly
+in a wide range of locations, including setting up access and configuring images and machine specs.
+The configuration options are described in more detail below.
+
+In some cases, a cloud provider may offer special features or unexpected quirks;
+known issues are described a the **[More Details for Specific Clouds](more-clouds.html)** page.
```
Comment (drigodwin) [inline guide/ops/locations/index.md]: How about:

In some cases, a cloud provider may offer special features or have unexpected quirks. Known issues such as these are described in **[More Details for Specific Clouds](more-clouds.html)**.

File: guide/ops/locations/index.md [hunk 2]
```diff
@@ -124,8 +129,9 @@ For more keys and more detail on the keys below, see
 - Specific Security Groups can be specified using `securityGroups`, as a list of strings (the existing security group names),
   or `inboundPorts` can be set, as a list of numeric ports (selected clouds only)
 
-- A specific existing key pair known at the cloud to use can be specified with `keyPair`
-  (selected clouds only)
+- A specific existing key pair known at the cloud to use for login can be specified with `keyPair`
+  (selected clouds only); if this `keyPair` does not correspond to your default `~/.ssh/id_rsa`
+  then you must typically also specify the `loginUser.privateKeyFile` to use with this `keyPair`
```
Comment (drigodwin) [inline guide/ops/locations/index.md]: An existing key pair, configured for login with the cloud can be specified with `keyPair` (selected clouds only). If this `keyPair` is not stored in your default `~/.ssh/id_rsa`, then you must typically also specify the `loginUser.privateKeyFile` which does contain this `keyPair`

**Comment by drigodwin** [comment]
This looks good, just two minor points. Thanks!

**Comment by ahgittin** [comment]
thanks @drigodwin , updated and merging

**Pull request merged**